File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n,sp,hs;
    
    do
    {
        printf("Height: ");
        n=GetInt();         
    }
    while(n<0 || n>23);
    
    
    for (int i=0;i<n;i++)       
    {
        for(sp=(n-1)-i;sp>0;sp--)        
            printf(" ");              
        
        for(hs=(n-1)-i; hs<=n;hs++)     
            printf("#");
        printf("\n");
    }
        
    
    return 0;
}



//  ##    i=0, (spaces= 2, hashes=2)     
// ###    i=1, (spaces =1, hashes=3)     
//####    i=2, (spaces =0, hashes=4)

File: pset6/Python_programs/mario.py
from cs50 import get_int

while True:
    print('Height: ', end='')  #n=4
    n= get_int()
    if(n>=0 and n<=23):
        break
    
for i in range(n):  #0,1,2,3
    print(('##' + '#'*i).rjust(n+1))


    
